model/dialogue/hmnet_model.py

```diff
@@ -24,10 +24,10 @@ class HMNetModel(SummModel):
     is_extractive = False
     is_neural = True
 
-    def __init__(self):
+    def __init__(self, **kwargs):
```

Comment:
Okay, this leaves almost no information for the users/other team members to know what to input here, hence it's not a good interface.

`**kwargs` is nothing more than a dictionary, when you pass in the args, items that have no direct match in other argument keys will be put into `**kwargs`

Instead, suppose you want to reveal one option (let's call it `some_arg`) and hide the others with `**kwargs`: 
```
def __init__(self, some_arg: some_type = some_default_value, **kwargs):
    kwargs["some_arg"] = some_arg
    # proceed to next lines
```

Reply:
Thanks for the great idea! I've modified the code